locust/main.py

```diff
@@ -242,6 +242,15 @@ def parse_options():
         help="sets the exit code to post on error"
     )
 
+    parser.add_argument(
+        '-s', '--stop-timeout',
+        action='store',
+        type=int,
+        dest='stop_timeout',
+        default=None,
+        help="number of seconds to wait for a taskset to complete an iteration before exiting. default is to terminate immediately."
```

Comment:
Alternative copy suggestion:
`Number of seconds to wait for a simulated user to complete any executing task before exiting. Default is to terminate immediately.`